package.json

```diff
@@ -2,17 +2,16 @@
   "name": "eslint-config-skyscanner",
   "version": "5.0.0-beta.2",
   "description": "Skyscanner's ESLint config.",
-  "main": "index.js",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": "^12.13.0",
+    "npm": "^6.12.0"
+  },
   "repository": {
     "type": "git",
     "url": "https://github.com/Skyscanner/eslint-config-skyscanner"
   },
-  "engines": {
-    "node": "^8.9.0",
-    "npm": "^6.4.1"
-  },
-  "author": "Matthew Davidson <matthew.davidson@skyscanner.net>",
-  "license": "Apache-2.0",
+  "author": "Backpack Design System <backpack@skyscanner.net>",
```

Comment:
Changing author to keep it aligned with other repos. Matt, forgive me 🙏 😅

Reply:
Nice 👍